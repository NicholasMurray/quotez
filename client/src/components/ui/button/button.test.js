import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';
import Button from '.';

test('it renders the default button correctly', () => {
  const tree = TestRenderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('it renders the disabled button correctly', () => {
  const tree = TestRenderer.create(<Button disabled />).toJSON();
  expect(tree).toMatchSnapshot();
});
