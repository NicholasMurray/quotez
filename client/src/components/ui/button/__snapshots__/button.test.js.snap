// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the default button correctly 1`] = `
.c0 {
  background-color: #66c7f4;
  border-radius: 5px;
  color: #2d3047;
  font-size: 1.5rem;
  font-weight: bold;
  margin: 2rem 0;
  padding: 1rem;
}

.c0:disabled {
  background-color: grey;
}

<button
  className="c0"
/>
`;

exports[`it renders the disabled button correctly 1`] = `
.c0 {
  background-color: #66c7f4;
  border-radius: 5px;
  color: #2d3047;
  font-size: 1.5rem;
  font-weight: bold;
  margin: 2rem 0;
  padding: 1rem;
}

.c0 true:disabled {
  background-color: grey;
}

<button
  className="c0"
  disabled={true}
/>
`;
